README.md

```diff
@@ -0,0 +1,20 @@
+### Backend
+For the backend, to start the server, run:
+
+```
+npm start --> starts the node express server
+```
+
+## Environmental Variables
+
+Backend (.env in server folder)
+
+Example:
+
+```
+# Server port
+PORT=3000
+
+# Prisma db connection
+DATABASE_URL=postgresql://janedoe:mypassword@localhost:5432/mydb?schema=sample
```

Comment:
@thaistcosta i am hosing the database on Heroku, as I didn't know if we'd have any problems running it on each of our local machines...